lib/setup-lib.bash

```diff
@@ -113,7 +113,8 @@ function installed_direnv() {
   local version=$1
   case "$version" in
     system | SYSTEM)
-      ASDF_DIRENV_BIN="$(run_cmd env ASDF_DIRENV_VERSION=system asdf which direnv)"
+      # Take only the first direnv that is not provided by asdf shims.
+      ASDF_DIRENV_BIN="$(which -a direnv | grep -v asdf | head -n 1)"
```

Comment:
`which` isn't necessarily installed. Because this is a bash script, `type -aP` would be better.